assets/src/activation-notice/app/components/dismiss.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Should we ignore this file from test coverage? It is a workaround. If not, can we test it somehow?

Reply:
Ignoring from coverage seems unnecessary. Things to test would be that it renders `null` & that it emits an event.

The actual functionality I think can be tested in an E2E test.